app/representers/proxy_representer.rb

```diff
@@ -31,7 +31,7 @@ class ProxyRepresenter < ThreeScale::Representer
   property  :sandbox_endpoint
   property  :api_test_path
   property  :api_test_success
-  property  :policies_config
+  property  :policies_config, exec_context: :decorator
```

Comment:
I thought that this is needed to override `#policies_config`. I'll try to remove this change and see if it still works.

Reply:
I must have done some mistake when triggering tests because this is indeed needed. Not sure where on the internets I found it. But without it `ProxiesControllerTest` and `proxy_spec.rb` fail on converting the Proxy object to JSON properly. The overriding of `#policies_config` seems ignored without it.